cmd/restic/cmd_prune.go

```diff
@@ -475,9 +475,9 @@ func prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedB
 	Verboseff("partly used packs:  %10d\n", stats.packs.partlyUsed)
 	Verboseff("unused packs:       %10d\n\n", stats.packs.unused)
 
-	Verboseff("to keep:   %10d packs\n", stats.packs.keep)
-	Verboseff("to repack: %10d packs\n", len(repackPacks))
-	Verboseff("to delete: %10d packs\n", len(removePacks))
+	Verboseff("to keep:      %10d packs\n", stats.packs.keep)
+	Verboseff("to repack:    %10d packs\n", len(repackPacks))
+	Verboseff("to delete:    %10d packs\n", len(removePacks))
 	if len(removePacksFirst) > 0 {
 		Verboseff("to delete: %10d unreferenced packs\n\n", len(removePacksFirst))
```

Comment:
Need three additional spaces here too :)

Reply:
I'm confused. I already added 3 spaces there, rebased it to squash it up, and force pushed it. The three spaces show up when I look at the commit?

Reply:
I think this comment refers to this line: "to delete: %10d unreferenced packs\n\n". That one belongs to the same group of output values.

Reply:
That makes sense. I've edited that line as well. Thanks!